components/automate-gateway/handler/authz_test.go

```diff
@@ -226,7 +154,7 @@ func TestIntrospectSome(t *testing.T) {
 	for name, tc := range cases {
 		t.Run(name, func(t *testing.T) {
 			ctx := auth_context.NewContext(
-				context.Background(), []string{"user:local:admin"}, []string{"project"}, "some:resource", "some:action", "AuthV1")
+				context.Background(), []string{"user:local:admin"}, []string{"project"}, "some:resource", "some:action", "Auth")
```

Comment:
What is `"Auth"`?

Reply:
Good question...found an answer, patched it here, and added further cleanup to https://github.com/chef/automate/issues/2807